catalogue/webapp/components/WorkSearchResults/WorkSearchResults.js

```diff
@@ -35,7 +35,7 @@ const WorkSearchResults = ({ works, worksRouteProps, apiProps }: Props) => {
                 id: result.id,
                 position: i,
                 resultWorkType: result.workType.label,
-                resultLanguage: result.language && result.language.label,
+                resultLanguage: result.languages.length > 0 && result.languages[0].label,
```

Comment:
The build is failing here because I'm sending the wrong type.

I don't understand our tracking especially well – is the correct fix to send the right single-language work type, or would it be better to send the complete languages list here?

(And if the former, what am I doing wrong with my current type?)

Reply:
Is this something that has been asked for on search relevance tracking? e.g. What languages are being selected in the search results?

Reply:
Good question. I know nothing about tracking, so I'm not sure. Would Tacey be the right person to ask?

Reply:
Let's leave it out as I am removing the relevancy tracking (should have said this first) - I will add it as part of the questions.

For info, we will be moving towards tracking a page / work view, and we will attach attributes of the work to the event there.